src/python/ensembl/databases/core_db_model.py

```diff
@@ -0,0 +1,997 @@
+from sqlalchemy import (Column, Index, Enum, ForeignKey, DateTime, Float, DECIMAL, text)
```

Comment:
We already have those source available here:  https://github.com/Ensembl/ensembl-py/blob/main/src/python/ensembl/core/models.py Is it possible to reuse ?

Reply:
Hi Marc. Sure that's possible, I have updated the code to use that file instead (imported from ensembl.core)